hicommonwealth/commonwealth
# 6683 detect enter for lists

## Link to Issue
Closes: #6683 

## Description of Changes
- Added a `onKeyDown` prop to the Quill Editor
- Fixed a secondary bug with the toolbar 
- Adds the ability to break out of lists

## "How We Fixed It"

- On `handleKeyDown` It matches the current list type by looking backwards in the editor and continuing the list on the event of `Enter`

## Test Plan
- Create any of the three lists
- Hit enter
- Ensure the lists continue 
- Hitting enter twice rapidly should break out of the list
